contrail-overlay.nix

```diff
@@ -253,9 +256,13 @@ in rec {
     };
   });
 
-  contrail50 = contrail41.overrideScope' (lself: lsuper: {
+  contrail50 = contrail41.overrideScope' (lself: lsuper: rec {
     contrailVersion = "5.0";
     contrailSources = callPackage ./sources-R5.0.nix { };
+    webui = callPackages ./pkgs/webui.nix { sources = contrailSources; };
+    test = lsuper.test // {
+      webui = callPackages ./test/webui.nix { contrailPkgs = lself; };
```

Comment:
`callPackage`

Reply:
Actually this *needs* to be `callPackages` otherwise hydra will not find it